notebooks/connecting_circles_with_pulled_station_id_data.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>do we have good reason to believe it's latin_1 encoding?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>